languages/persian/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = false
```

Comment:
This should be `true`, since we are going to use this code in production

Reply:
I've done the requested changes, if that's fine I resolve this conversation.